packages/shared/src/components/squads/Comment.tsx

```diff
@@ -203,7 +231,7 @@ export function SquadComment({
               className="btn-primary-cabbage"
               type="submit"
               loading={isLoading}
-              disabled={!commentary || isLoading || isCheckingUrl}
+              disabled={!commentary || isLoading || isPreviewLoading}
```

Comment:
I'm also pretty sure we block it if preview would fail for whatever reason. not sure this captures that (just want to make sure)

Reply:
Oh, are not gonna allow anymore if there is no title and image from the preview API?

Reply:
I believe so, if this one can't return the actual scraper won't be able to.
But maybe triple check with Sab on this.